### DISCUSSION TITLE
Is the project dead

### DISCUSSION THREAD

**Discussion by gun4qmm7h** [Chooloo/koler#529] [Q&A] [answered]
Koler hasn't resived an update in forever and I really injoy the app but it does have bugs that make it unreliable. I was waiting months hoping that the updates will make the app more stable but sadly the last update happen in February. Has the developer given up or just taking a break

**Comment by Mylloon** [comment]
As a replacement, Fossify apps have a [phone app](https://github.com/FossifyOrg/Phone)

**Comment by inson1** [comment]
yea 8 months without commit ....

**Comment by roeiedri** [comment, marked as answer]
Hey, it's not dead yet :) 
Been working slowly on a new Compose version meaning changing the entire code
It's going well and hopefully we'll see a working commit soon @Mylloon @inson1 @gun4qmm7h

**Reply by gun4qmm7h** [reply]
Reading this made me smile. I really enjoy the app

**Reply by gun4qmm7h** [reply]
Any idea when the new app will come out?

**Reply by gun4qmm7h** [reply]
@roeiedri any update on the phone app? Is there a beta version that we can test to help out?

**Reply by JoeJoeTV** [reply]
@roeiedri Is this still happening? Can I follow it somewhere? I like the app, but I've been having some issues lately, so I can't use it as my default dialer app currently.

**Comment by Queuecumber** [comment]
So I think we can confirm the project is dead now right?
